src/xrpld/app/tx/detail/InvariantCheck.cpp

```diff
@@ -499,7 +501,8 @@ AccountRootsDeletedClean::finalize(
     // be logged
     [[maybe_unused]] bool const enforce =
         view.rules().enabled(featureInvariantsV1_1) ||
-        view.rules().enabled(featureSingleAssetVault);
+        view.rules().enabled(featureSingleAssetVault) ||
+        view.rules().enabled(featureLendingProtocol);
```

Comment:
If featureInvariantsV1_1 is enabled, but one node hasn’t upgraded to the rippled version that includes lending while another node has (and therefore runs the new invariant checks), wouldn’t they end up producing different transaction results? (I guess it would be okay since invaraints are meant to be defensive checks)